### PR TITLE
Train any model from the hub

### PR DESCRIPTION
**Pull request by abhi1thakur**

### CODE REVIEW

File: src/autonlp/autonlp.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def _login_from_conf(self):
                         self.orgs = conf_json["orgs"]
                         self.token = conf_json["token"]
 
-    def create_project(self, name: str, task: str, language: str, max_models: int):
+    def create_project(self, name: str, task: str, language: str, max_models: int, hub_model: str = None):
```
Comment (SBrandeis) [inline src/autonlp/autonlp.py:76]: Might be time to add proper docstrings in the package!

File: src/autonlp/autonlp.py [hunk 2]
```diff
@@ -88,7 +88,13 @@ def create_project(self, name: str, task: str, language: str, max_models: int):
             "username": self.username,
             "proj_name": name,
             "task": task_id,
-            "config": {"version": 0, "patch": 1, "language": language, "max_models": max_models},
+            "config": {
+                "version": 0,
+                "patch": 1,
```
Comment (SBrandeis) [inline src/autonlp/autonlp.py:93]: Not needed anymore I believe

**Review by SBrandeis — COMMENTED** [review]
LGTM, the docstrings should be added in another PR
Let's wait for the backend feature is available before merging

File: src/autonlp/autonlp.py (hunk 1 above)
Comment (abhi1thakur) [reply src/autonlp/autonlp.py:76]: maybe we should add all docstrings in a single or. this is limited to cli and cli has hints.

File: src/autonlp/autonlp.py (hunk 2 above)
Comment (abhi1thakur) [reply src/autonlp/autonlp.py:93]: fixed

**Pull request merged**